lib/terminal-view.js

```diff
@@ -106,6 +108,11 @@ class TerminalView extends View {
       localStorage.set('popout-emulator:font-size', this.currentFontSize())
       localStorage.set('popout-emulator:font-family', $(this.emulator.element).css('font-family'))
 
+      let xterm = path.join(app.getAppPath(), 'node_modules', 'xterm', 'dist', 'xterm.css')
+      let fullscreen = path.join(app.getAppPath(), 'node_modules', 'xterm', 'dist', 'addons', 'fullscreen', 'fullscreen.css')
```

Comment:
@joshrowley `app.asar` worked! that's what these paths are based on. I just picked the commits from `learn-co/learn-ide-3` since this is also broken in this repo.